src/components/Text/styled.tsx

```diff
@@ -0,0 +1,51 @@
+import styled, { css } from 'styled-components';
+import { em } from 'polished';
+
+import { boxSizing } from '../../modules/box-sizing';
+import { styleless } from '../Styleless';
+
+export const sizes = [
+  'micro',
+  'tiny',
+  'small',
+  'reduced',
+  'normal',
+  'increased',
+  'huge',
+  'giant',
+] as const;
+export type Size = typeof sizes[number];
+
+export const weights = ['light', 'regular', 'bold'] as const;
+export type Weight = typeof weights[number];
+
+export const Styled = styled.div<{ color?: string; size: Size; weight?: Weight }>`
+  ${styleless};
+  ${boxSizing};
+
+  color: ${({ theme }) => theme.honeycomb.color.text.normal};
+  font-weight: 400;
```

Comment:
Do we have common size values for these here?

Reply:
No. 😦